src/custom_api_response.py

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# heritedcode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/heritedcode for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+
+class CustomApiResponse:
+    """
```

Comment:
Why would you need such a class?  And why split your code in two python files? One file is simpler and easier to read in this case. Python is not like Java. We like things simple.
Try running this:
```
$ python
>>> import this
```

Reply:
> Why would you need such a class? And why split your code in two python files? One file is simpler and easier to read in this case. Python is not like Java. We like things simple. Try running this:
> 
> ```
> $ python
> >>> import this
> ```

Actually @TG1999 suggested me to have our own class and then while returning response from a function make new object of this class and set those json attributes as this class attribute.
But actually i also didn't got why is it need if we are getting json in response from SWH api which is standard right??